blocks/init/src/Blocks/custom/map/components/map-editor.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { checkAttr } from '@eightshift/frontend-libs/scripts';
+import { Interactions, Layers, MapInteraction, OpenLayersMap, Controls, MapControl, processMapLayer } from './map-components';
+import manifest from '../manifest.json';
+
+export const MapEditor = ({ attributes }) => {
+	const {
+		blockClass,
+	} = attributes;
+
+	const mapLayers = checkAttr('mapLayers', attributes, manifest);
+	const mapCenterLat = checkAttr('mapCenterLat', attributes, manifest, true);
+	const mapCenterLon = checkAttr('mapCenterLon', attributes, manifest, true);
+	const mapZoom = checkAttr('mapZoom', attributes, manifest);
+	const mapInteractions = checkAttr('mapInteractions', attributes, manifest);
+	const mapControls = checkAttr('mapControls', attributes, manifest);
+
+	return (
+		<OpenLayersMap center={[mapCenterLat ?? 46.314045, mapCenterLon ?? 16.352532].reverse()} zoom={mapZoom} blockClass={blockClass}>
```

Comment:
Don't want to have the values pre-filled (so you can just paste in the value you want first, instead of having to delete the default value first), but also didn't want the default view to be the middle of the ocean, so I picked Varaždin 😄 

But I can add it, it's not a problem